File: java-main/java-main/Whileloop/p8.java
class Demo{
	public static void main(String[]args){

		int N=135;
		int mul=1;

		while(N!=0){
			int rem=N%10;
			mul=mul*rem;
			N=N/10;
		}
		System.out.println(mul);
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/practical2/p8.c
#include<stdio.h>
void main(){

	int num;
	int i=1;

	int rem=0,mul=1;

	printf("Enter the number\n");
	scanf("%d",&num);

	while(i<=num){
		rem=num%10;
		num=num/10;
		mul=mul*rem;
		i++;
	}
	printf("mul is :%d\n",mul);
}

